docs/source/checkpoints/intro.rst

```diff
@@ -20,6 +20,26 @@ With sharded model weights, you can save and load the state of your training scr
 
 NeMo supports the distributed (sharded) checkpoint format from Megatron-Core. In Megatron-Core, it supports two backends: Zarr-based and PyTorch-based.
 
+Quantized Checkpoints
+---------------------
+
+NeMo also offers :doc:`Post-Training Quantization <../nlp/quantization>` workflow to convert regular ``.nemo`` models into a `TensorRT-LLM checkpoint <https://nvidia.github.io/TensorRT-LLM/architecture/checkpoint.html>`_ conventionally referred to as ``.qnemo`` checkpoints in NeMo. Such a checkpoint can be used with `NVIDIA TensorRT-LLM library <https://nvidia.github.io/TensorRT-LLM/index.html>`_ for efficient inference.
+
+Much as in the case of ``.nemo`` checkpoints, a ``.qnemo`` checkpoint is a tar file that bundles the model configuration given in ``config.json`` file and ``rank{i}.safetensors`` files storing model weights for each rank separately. Additionally a ``tokenizer_config.yaml`` file is saved which is just ``tokenizer`` section from ``model_config.yaml`` file from the original NeMo model. This configuration file defines a tokenizer for the model given.
```

Comment:
.qnemo would not support distributed checkpoint format? i.e. you saved with world_size 2 and have to load with world_size 2?

Reply:
One thing to clarify is that these `config.json` + `rank{i}.safetensors` output is a TRT-LLM checkpoint. This should not be confused by distributed checkpoint in Nemo sense.

Anyway, the feature you asked for is not available in TRT-LLM currently. So to build a TRT-LLM engine with world_size=2 one needs to calibrate/quantize model to TRT-LLM checkpoint with world_size=2 and provide this as the input to `trtllm-build` command. In other words, world_size cannnot be changed at engine build.